apps/arweave/src/ar_http_iface_server.erl

```diff
@@ -121,6 +121,9 @@ name_route([<<"vdf">>, <<"session">>]) ->
 	"/vdf/session";
 name_route([<<"vdf2">>, <<"session">>]) ->
 	"/vdf2/session";
+name_route([<<"vdf3">>, <<"session">>]) ->
```

Comment:
Eventually we can get rid of the old formats, right? We need to introduce them mostly to handle the upgrade from one release to the next, but once everyoneis on the same release we can retire the old endpoints, right?

E..g. we can retire format 1, /vdf, and /vdf/session whenever we're ready, right? (since it's been a long time and at least 1 hard fork since we introduce them)

Reply:
Makes sense

Reply:
Logged for later: https://github.com/ArweaveTeam/arweave-dev/issues/513